File: official/docs/golang/v4/shipments/label.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func label() {
	client := easypost.New("EASYPOST_API_KEY")

	shipment, _ := client.GetShipmentLabel("shp_...", "ZPL")

	fmt.Println(shipment)
}

File: official/docs/csharp/v4/shipments/label.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Models.API.Shipment shipment = await client.Shipment.Retrieve("shp_...");

            await shipment.GenerateLabel("ZPL");

            Console.WriteLine(JsonConvert.SerializeObject(shipment, Formatting.Indented));
        }
    }
}
